### ISSUE TITLE
To-do list check marks fail WCAG

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5-list#146]
![image](https://user-images.githubusercontent.com/1099479/63262022-34930500-c285-11e9-863a-b01276c0c891.png)

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=30D23F

[Should be #1D8727 to pass AA for normal text](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=1D8727).

![image](https://user-images.githubusercontent.com/1099479/63262153-80de4500-c285-11e9-8c12-cf359b5b5c1b.png)

cc @dkonopka

**Comment by dkonopka** [comment]
Current background of checkbox `hsl(120, 100%, 42%)` has 1.97 contrast ratio
Your proposed color has 4.61 ratio.

But, it's a kind of graphical object so shouldn't we pass only `3.0` ratio? E.g: `#26AB33`

![Screen Shot 2019-08-19 at 13 39 41](https://user-images.githubusercontent.com/10219857/63262677-e121b680-c286-11e9-9a66-38a67f045665.png)

**Comment by oleq** [comment]
Alright. Let's settle this at 3:1.
